eessi/testsuite/hooks.py

```diff
@@ -34,6 +34,27 @@ def assign_default_num_cpus_per_node(test: rfm.RegressionTest):
     log(f'default_num_cpus_per_node set to {test.default_num_cpus_per_node}')
 
 
+def assign_default_num_gpus_per_node(test: rfm.RegressionTest):
```

Comment:
to be consistent with other functions that are local to `hooks.py`: should the function name start with `_`?